### ISSUE TITLE
Representation of sensor calibration further information

### ISSUE THREAD

**Issue by JuangaSocib** [OceanGlidersCommunity/OG-format-user-manual#45]
Hi mates,

I think it worth to add a new variable attribute next to SENSOR_CALIBRATION_DATE aiming to represent a PID (eg. PURL, DOI, handle ID or whatever), to link the calibration sheets in use for the sensor measuring the variable. This information would complement the calibration date for users to explore further details regarding data acquisition. Currently at SOCIB we are developing a pilot netcdf (for the moment based on the SOCIB format) capable of represent this link and provide a landing page for the calibration sheets.

**Comment by justinbuck** [comment]
Hi Juanga, I agree including such information in the file would the useful to users, I think it needs a bit more design work though (to cover pre and post deployment calibrations, the different sensors, etc), e.g.,
SENSOR_CALIBRATION_DATE (N_CALIB) - the data of the calibration event
SENSOR_CALIBRATION_PARAM/SENSOR (N_CALIB) - the parameter being calibrated
SENSOR_CALIBRATION_LINK (N_CALIB) - link to calibration sheets

**Comment by soerenthomsen** [comment]
Hi @JuangaSocib thanks for your great input here! I just invited you to the GitHub community.

**Comment by vturpin** [comment]
I also agree. This could be helpfull to be able to access the cal sheet if needed.
Can someone make a PR in that sense ?

**Comment by castelao** [comment]
@JuangaSocib , thanks for your support on #103. With the approval of #103, may I close this issue, or is there anything still missing not addressed by #103?

**Comment by JuangaSocib** [comment]
Hi @castelao, I just closed PR #69, however I'm missing the SENSOR_CALIBRATION_LINK information. in the new version of the format. To my understanding this might correspond to a new global attribute within each sensor group (e.g. calibration_link). I will open a new PR if you agree.

**Comment by castelao** [comment]
Yes, good catch @JuangaSocib . Please, open a new PR to add that. I suggest including @justinbuck in the discussions, he might have good advice on using PIDs or other alternatives for DOIs to track it.

Let's call it 'group attributes' to clearly distinguish from 'global attributes'?

Thank you!

**Comment by vturpin** [comment]
#110
